Makefile

```diff
@@ -40,8 +40,9 @@ $(BINARY):
 	CGO_ENABLED=0 $(GO) build -a -installsuffix cgo -ldflags="-X main.VERSION=${VERSION}" -o $@ github.com/oauth2-proxy/oauth2-proxy/v7
 
 DOCKER_BUILD_PLATFORM ?= linux/amd64,linux/arm64,linux/ppc64le,linux/arm/v6
+DOCKER_BUILD_RUNTIME_IMAGE ?= alpine:3.15
 DOCKER_BUILDX_ARGS ?=
-DOCKER_BUILDX := docker buildx build ${DOCKER_BUILDX_ARGS} --build-arg VERSION=${VERSION}
+DOCKER_BUILDX := docker buildx build ${DOCKER_BUILDX_ARGS} --build-arg VERSION=${VERSION} --build-arg RUNTIME_IMAGE=${DOCKER_BUILD_RUNTIME_IMAGE}
 DOCKER_BUILDX_X_PLATFORM := $(DOCKER_BUILDX) --platform ${DOCKER_BUILD_PLATFORM}
 DOCKER_BUILDX_PUSH := docker buildx build --push ${DOCKER_BUILDX_ARGS} --build-arg VERSION=${VERSION}
```

Comment:
The new build args will need to be added here as well else the push won't recognise the cached builds/won't have the correct RUNTIME_IMAGE

Reply:
I moved it to `DOCKER_BUILDX_ARGS`, as it's used in both `DOCKER_BUILDX` and `DOCKER_BUILDX_PUSH`